packages/bpk-scrim-utils/src/withScrimmedPortal-test.tsx

```diff
@@ -17,7 +17,7 @@
  */
 
 import '@testing-library/jest-dom';
-import { render, within, screen } from '@testing-library/react';
+import { render, within, screen, isInaccessible } from '@testing-library/react';
```

Comment:
is `isInaccessible` used somewhere?

Reply:
Ohh no, I'd added it, but then removed it
I'll remove this, I thought the linting would do that 😅